src/Tribe/Cache.php

```diff
@@ -536,4 +536,94 @@ public function data_size_over_packet_size( $value ) {
 		// If the size of the string is above 90% of the database `max_allowed_packet` setting, then it should not be written to the db.
 		return $size > ( $feature_detection->get_mysql_max_packet_size() * .9 );
 	}
+
+	/**
+	 * Returns a transient that might have been stored, due ot its size, in chunks.
+	 *
+	 * @since TBD
+	 *
+	 * @param string               $id                 The name of the transients to return.
+	 * @param string|array<string> $expiration_trigger The transient expiration trigger(s).
+	 *
+	 * @return false|mixed Either the transient value, joined back into one, or `false` to indicate
+	 *                     the transient was not found or was malformed.
+	 */
+	public function get_chunkable_transient( $id, $expiration_trigger = '' ) {
+		$transient = $this->get_id( $id, $expiration_trigger );
+
+		if ( wp_using_ext_object_cache() ) {
+			return get_transient( $transient );
+		}
+
+		$chunks = [];
+		$i      = 1;
+		do {
+			$chunk_transient            = $transient . '_' . $i++;
+			$chunk                      = get_transient( $chunk_transient );
+			$chunks[ $chunk_transient ] = (string) $chunk;
+		} while ( ! empty( $chunk ) );
+
+		if ( empty( array_filter( $chunks ) ) ) {
+			return false;
+		}
+
+		try {
+			$data          = implode( '', $chunks );
+			$is_serialized = preg_match( '/^[aO]:\\d+:/', $data );
+			$unserialized  = maybe_unserialize( implode( '', $chunks ) );
+			if ( is_string( $unserialized ) && $unserialized === $data && $is_serialized ) {
+				// Something was messed up.
+				return false;
+				return true;
+			}
```

Comment:
Sorry looks like this one was placed on the wrong place.

```suggestion
			if ( is_string( $unserialized ) && $unserialized === $data && $is_serialized ) {
				// Something was messed up.
				return false;
			}
			
			return true;
```